.storybook/BoardGameContextMock.js

```diff
@@ -1,57 +1,62 @@
 import React from "react";
 import { boolean, number, text, select } from "@storybook/addon-knobs";
 import { action } from "@storybook/addon-actions";
-import { BoardGameContext } from "contexts/BoardGameContext";
+import { BoardGameProvider } from "contexts/BoardGameContext";
 
 const ref = "Game Context";
 
-const BoardGameContextMock = ({ children }) => (
-  <BoardGameContext.Provider
-    value={{
-      G: {
-        remainingSeconds: number(
-          "Remaining seconds",
-          100,
-          {
-            range: true,
-            min: 0,
-            max: 120,
-            step: 1,
-          },
-          ref
-        ),
-        playersData: {
-          "0": {},
-          "1": { isActive: true },
-        },
+const BoardGameContextMock = ({ children }) => {
+  return (
+    <BoardGameProvider
+      G={{
+        timePerTurn: 120,
+        turnEndTime:
+          Math.floor(new Date().getTime() / 1000) +
+          number(
+            "Turn End Time",
```

Comment:
I think "Remaining seconds" was a good description for this knob

Reply:
👍